operator/validator/controller.go

```diff
@@ -631,7 +635,10 @@ func (c *controller) CommitteeActiveIndices(epoch phase0.Epoch) []phase0.Validat
 	return indices
 }
 
-func (c *controller) AllActiveIndices(epoch phase0.Epoch) []phase0.ValidatorIndex {
+func (c *controller) AllActiveIndices(epoch phase0.Epoch, afterMetadata bool) []phase0.ValidatorIndex {
```

Comment:
The `afterMetadata` variable is named `afterValidatorSetup`/`afterValidatorsSetup` in interfaces, should they have the same name?